### PR TITLE
Adds timeout on update check

### PR DESCRIPTION
**Pull request by bragagia**
It solves #274

### CODE REVIEW

**Comment by Soulou** [comment]
Disagreed, please add a timeout in the method `getLastVersion`

**Comment by bragagia** [comment]
If so, we need to create new goroutine inside `getLastVersion`, is that really useful ?

**Comment by Soulou** [comment]
No need to add a goroutine, you can set a timeout on the HTTP client :-)

**Comment by bragagia** [comment]
Done :)

File: update/check.go [hunk 1]
```diff
@@ -37,6 +40,11 @@ func Check() error {
 	}
 
 	<-gotLastVersion
+
+	if gotAnError {
+		fmt.Println("Timeout when connecting on scalingo server.")
```
Comment (Soulou) [inline update/check.go]: Why?

Comment (bragagia) [inline update/check.go]: I did'nt noticed that this was called on every command. Removed ;)

**Pull request merged**